design/ceph/object-store.md

```diff
@@ -95,51 +97,19 @@ The RGW service can be configured to listen on both http and https by specifying
     instances: 1
 ```
 
-### Realms, Zone Groups, and Zones
-
-By default, the object store will be created independently from any other object stores and replication to another object store will not be configured. This done by creating a new realm, zone group, and zone all with the name of the new object store. The zone group and zone are tagged as the `master`. If this is the first object store in the cluster, the realm, zone group, and zone will also be marked as the default.
+### Zone
```

Comment:
Just specifying the zone won't be enough as rook will create a different realm and zonegroup to use with this object store and we won't have replication.
The user must supply the realm, zonegroup and zone names (as were named in the CRs)
for example two RGWs:
First will have:
realm:
  name: myrealm
zonegroup:
  name: myzonegroup
zone:
    name: zone-a
Second will have:
realm:
  name: myrealm
zonegroup:
  name: myzonegroup
zone:
    name: zone-b

How about changing this section title to multisite?

Reply:
> Just specifying the zone won't be enough as rook will create a different realm and zonegroup to use with this object store and we won't have replication.

why? the zone should hold the definition of the zonegroup
automatically creating zone, zonegroup and realm is only for the case where `zone` is not provided in the object store

> The user must supply the realm, zonegroup and zone names (as were named in the CRs)
> for example two RGWs:
> First will have:
> realm:
> name: myrealm
> zonegroup:
> name: myzonegroup
> zone:
> name: zone-a
> Second will have:
> realm:
> name: myrealm
> zonegroup:
> name: myzonegroup
> zone:
> name: zone-b
> 
> How about changing this section title to multisite?

Reply:
We have decided to not have the user specify the zone, zonegroup, realm in the object-store config. The operator will be able to figure out which zonegroup, the zone belongs to, and which realm the zonegroup belongs to. 

This same line of thinking goes for specifying the realm and zonegroup for the zone CR as well.